lib/releases.sh

```diff
@@ -30,8 +30,11 @@ function get_latest_release() {
 CAPM3RELEASEPATH="${CAPM3RELEASEPATH:-https://api.github.com/repos/${CAPM3_BASE_URL:-metal3-io/cluster-api-provider-metal3}/releases}"
 CAPIRELEASEPATH="${CAPIRELEASEPATH:-https://api.github.com/repos/${CAPI_BASE_URL:-kubernetes-sigs/cluster-api}/releases}"
 
-export CAPIRELEASE="${CAPIRELEASE:-$(get_latest_release "${CAPIRELEASEPATH}" "v0.3.")}"
-
+if [ "${CAPI_VERSION}" == "v1alpha3" ]; then
```

Comment:
Are we planning to support CAPI v1alpha3 after v1a4 is released? And if we do, how long?

Reply:
Yes we are just defaulting the capi v1a4, we will support both version.